lib/dotcom_web/components/system_status/commuter_rail_timetable_status.ex

```diff
@@ -0,0 +1,159 @@
+  @moduledoc """
+  Component displaying the status of a specific commuter rail line.
+  """
+
+  use DotcomWeb, :component
+
+  import DotcomWeb.Components, only: [bordered_container: 1, unstyled_accordion: 1]
+  import DotcomWeb.Components.Alerts, only: [embedded_alert: 1]
+  import DotcomWeb.Components.SystemStatus.StatusLabel, only: [status_label: 1]
+
+  alias Alerts.Alert
+
+  attr :status, :map
+
+  def commuter_rail_timetable_status(assigns) do
+    ~H"""
+    <.bordered_container hide_divider>
+      <:heading>Current Status</:heading>
+      <div class="border-b-xs border-gray-lightest mt-4">
+        <.rows_for_status status={@status} />
+      </div>
+    </.bordered_container>
+    """
+  end
+
+  defp rows_for_status(%{status: :normal} = assigns) do
+    ~H"""
+    <.row>
+      <.status_label status={:normal} description="Normal Service" />
+    </.row>
+    """
+  end
+
+  defp rows_for_status(%{status: :no_scheduled_service} = assigns) do
+    ~H"""
+    <.row>
+      <.status_label status={:no_scheduled_service} description="No Scheduled Service" />
+    </.row>
+    """
+  end
+
+  defp rows_for_status(assigns) do
+    ~H"""
+    <.service_alert_rows alerts={@status.service_alerts} />
+    <.train_impact_rows impacts={@status.cancellations} effect={:cancellation} />
+    <.train_impact_rows impacts={@status.delays} effect={:delay} />
+    """
+  end
+
+  defp service_alert_rows(assigns) do
+    ~H"""
+    <div :for={alert <- @alerts} class="">
+      <.unstyled_accordion
+        summary_class="flex items-center border-t-xs border-gray-lightest hover:bg-brand-primary-lightest cursor-pointer group/row py-3 px-1"
+        chevron_class="fill-gray-dark px-2 flex items-center"
+      >
+        <:heading>
+          <div class="grow">
+            <.status_label status={alert.effect} description={Alert.human_effect(alert)} />
+          </div>
+        </:heading>
+        <:content>
+          <.embedded_alert alert={alert} />
+        </:content>
+      </.unstyled_accordion>
+    </div>
+    """
+  end
+
+  defp train_impact_rows(%{impacts: []} = assigns), do: ~H""
+
+  defp train_impact_rows(assigns) do
+    human_effect = Alert.human_effect(%Alert{effect: assigns.effect})
+    count = Enum.count(assigns.impacts)
+
+    description =
+      if count == 1 do
+        "1 #{human_effect}"
+      else
+        "#{count} #{Inflex.pluralize(human_effect)}"
+      end
+
+    assigns = assigns |> assign(:description, description)
+
+    ~H"""
+    <.row>
+      <.status_label status={@effect} description={@description} />
+    </.row>
+
+    <div>
+      <.unstyled_accordion
+        :for={impact <- @impacts}
+        class="first:-mt-1"
+        summary_class="flex items-center grow pl-7.5 py-1 hover:bg-brand-primary-lightest"
+        chevron_class="fill-gray-dark px-2 flex items-center"
+      >
+        <:heading>
+          <span class="grow"><.trip_info_heading trip_info={impact.trip_info} /></span>
```

Comment:
<img width="458" alt="image" src="https://github.com/user-attachments/assets/1370a66e-1444-472e-ad22-1e2160828ccd" />
 FYI the right caret is a little too far right. I think this could also benefit from a little more padding below the text.